http.go

```diff
@@ -89,7 +89,7 @@ func statusIndex(w http.ResponseWriter, r *http.Request) {
 	// Set logs
 	if  data.SupervisorState != "" || !data.SupervisorConnected {
 		var buf bytes.Buffer
-		var re = regexp.MustCompile(`\[\d+m`)
+		var re = regexp.MustCompile(`\x1b\[\d+m`)
```

Comment:
This one is fun, the escape character in the original code isn't even shown in my browser:

![image](https://github.com/user-attachments/assets/115e1149-9007-4b02-b4e9-159ab557764b)

It is though when I copy the code e.g. to gedit:

![image](https://github.com/user-attachments/assets/86a5c38c-9e36-4f95-a8e3-8cfdcb196941)

Reply:
Right, so that linter check definitly make sense :sweat_smile: :+1: